### ISSUE TITLE
Groupmanager OP's override

### ISSUE THREAD

**Issue by ShadedPL** [mbax/NameTags#8]
When having OP the plugin sees the Groupmanager permissions as if the player had '*' as it's permission and showing random colours after every reload. I barely have any permission assinged.

**Comment by mbax** [comment]
If your permissions plugin treats OP as having all nodes, even ones defaulted to false (such as in NameTags) that is a bug with your permissions plugin. NameTags handles OP correctly, and only grants the reload permission. :)

**Comment by ShadedPL** [comment]
But it is not handled as op by other plugins. I have for example a plugin called PermaPotions (http://dev.bukkit.org/bukkit-plugins/permapotions/) and this one does not see OP's as full permission guys cause otherwise it would give me all potioneffects possible. So if the other plugin works fine and this one isn't I think it's rather a bug with THIS plugin. I'm just saying :) I can't work with this plugin when it is working like that. I had never before problems with my permission nodes exept with NameTags.

**Comment by mbax** [comment]
Here's an easy test!

Set up a test server with **only** TagAPI and NameTags. Give yourself OP. Do you get random colors?

**Comment by ShadedPL** [comment]
Here is the thing. I've figured it out what's going on. When having no NameTags permission and I am OP my nick is white but for instance when i give myself only one permission of NameTags it starts beeing random.
Deopping myself makes it work normal.

Like you said I had only 3 plugins: TagAPI, NameTags and Groupmanager.

**Comment by mbax** [comment]
I said only TagAPI and NameTags. You're still adding in GroupManager. Just do NameTags and TagAPI and OP and see what happens.

**Comment by ShadedPL** [comment]
Nothing happens, the nicknames are white. It's an issue with the way NameTags reads Groupmanagers permissions system.

**Comment by mbax** [comment]
NameTags doesn't read GroupManager's anything. NameTags isn't aware of what permissions plugin  you are running, because it just asks Bukkit "Does ShadedPL have this permission node?" and gets a reply. :)

GroupManager, Privileges, PermissionsBukkit, zPermissions, etc. all tell Bukkit "ShadedPL has these permission nodes." When you give yourself "meow.meow.meow" in GroupManager, GroupManager tells Bukkit "ShadedPL has meow.meow.meow". I can then write a plugin to ask "Does ShadedPL have meow.meow.meow?" and Bukkit will reply "Yes!" without my plugin ever knowing what plugin it was talking to.

In short, this is a GroupManager problem not a NameTags problem. :)

**Comment by ShadedPL** [comment]
But still I don't have this problem with other plugins and Groupmanager. And the weirdest part is the fact that when having one of NameTags permissions and beeing OP makes it so that I have all the NameTag permissions. When having no NameTag permission and OP the nicknames are just white, and no Randomness is occuring. I'm kinda confused but ok, I'll take your world for it. Gonna give this some tests.
EDIT: when doing the same with for example PermaPotions the problem does not occur. It shoud give me all permissions when giving myself just one but it doesn't. That confuses me just more :P

EDIT#2: NOW I'm confused... It started working normally and I don't know why... :D
